spec/datadog/tracing/contrib/sidekiq/client_tracer_spec.rb

```diff
@@ -27,7 +27,7 @@
     expect(span.get_tag('sidekiq.job.queue')).to eq('default')
     expect(span.status).to eq(0)
     expect(span).to be_root_span
-    expect(span.get_metric('_dd.measured')).to be_nil
+    expect(span.get_metric('_dd.measured')).to not_be nil
```

Comment:
Prefer stronger assertion `.to eq(...)`, instead of `.to not_be nil`.

Reply:
Fixed